drivers/clock_control/clock_control_sama7g5_sckc.c

```diff
@@ -51,19 +51,15 @@ static int sckc_get_rate(const struct device *dev,
 			 uint32_t *rate)
 {
 	ARG_UNUSED(dev);
+	ARG_UNUSED(sys);
 
+	bool sel_xtal = (sckc_reg->SCKC_CR & SCKC_CR_TD_OSCSEL_Msk) ==
+			SCKC_CR_TD_OSCSEL(SCKC_CR_TD_OSCSEL_XTAL_Val) ? true : false;
```

Comment:
Already a boolean result

```suggestion
	bool sel_xtal = (sckc_reg->SCKC_CR & SCKC_CR_TD_OSCSEL_Msk) ==
			SCKC_CR_TD_OSCSEL(SCKC_CR_TD_OSCSEL_XTAL_Val);
```

Reply:
Removed, thanks.